cmd/vic-machine/configure/configure.go

```diff
@@ -85,6 +105,59 @@ func (c *Configure) Flags() []cli.Flag {
 			Destination: &c.upgrade,
 			Hidden:      true,
 		},
+		cli.BoolFlag{
+			Name:        "regenerate-certs",
```

Comment:
I'll just change it to `--no-tlsverify` to make it match `create`. I actually think the name of the flag is bad in `create`, and I don't necessarily think that we should stick to strictly keeping the same flags in `create` and `configure` as their meanings can change with the context of reconfiguring the VCH. Obviously we want to keep them as close as possible for ease of use, but changing this to `--no-tlsverify` will actually hide the real purpose of the flag.